UNRELEASED.md

```diff
@@ -0,0 +1,6 @@
+**Changed:**
+- Deleted the following deprecated components:
+  - `bpk-component-navigation-stack`
+  - `PortalV1` (in `bpk-react-utils`)
+
+- Dropped support for React versions < 17. No new functionality is introduced, but from this point forwards it should not be assumed React 16 will work.
```

Comment:
As this is a big jump, particularly as there are still some React < 16 we should probably mark this as a breaking change as its officially dropping support

Reply:
Hm ok good point 👍